### ISSUE TITLE
direct install in windows,raise this error when test .what need to do?

### ISSUE THREAD

**Issue by x779250919** [TianZerL/pyanime4k#4]
~\AppData\Roaming\Python\Python37\site-packages\pyanime4k\wrapper\acwrapper.py in 
\#Add current path to trusted DLL directory in Windows
 if platform.system() == "Windows":
--->  os.add_dll_directory(curr_path)
 c_ac = ctypes.cdll.LoadLibrary(

AttributeError: module 'os' has no attribute 'add_dll_directory'

**Comment by TianZerL** [comment]
Temporary fix. Comment out error lines 
```
if platform.system() == "Windows":
    os.add_dll_directory(curr_path)
```
but, module 'os' has no attribute 'add_dll_directory'? This is a system function of Python for Windows, I guess your Python may not be fully installed.

**Comment by TianZerL** [comment]
This will be fixed in new version

**Comment by x779250919** [comment]
I use python3.7.3 by anaconda,  os really has no 'add_dll_directory'.
is error with dll? have to manual install?

**Comment by TianZerL** [comment]
No, everything is ok, `add_dll_directory` is added by python3.8, I didn't notice that.

**Comment by TianZerL** [comment]
I will fix it soon, now you just need to comment out error lines
```
if platform.system() == "Windows":
    os.add_dll_directory(curr_path)
```

**Comment by x779250919** [comment]
......thanks your quick

**Comment by TianZerL** [comment]
It is actuall for making sure python can find out the dll files, but in later versions, I did it in a better way, so it ok to delete those two lines.

**Comment by TianZerL** [comment]
Please install the newest version by pip.
